Source/player.cpp

```diff
@@ -3039,8 +3039,12 @@ void ApplyPlrDamage(int pnum, int dam, int minHP /*= 0*/, int frac /*= 0*/, int
 	int totalDamage = (dam << 6) + frac;
 	if (totalDamage > 0 && player.pManaShield) {
 		int8_t manaShieldLevel = player._pSplLvl[SPL_MANASHIELD];
+		int dmgReduction = 24;

```

Comment:
It'd be good to extract this to a function, so that we can later display it in the spell book.